packages/forma-36-react-components/src/components/Button/Button.stories.tsx

```diff
@@ -318,3 +318,39 @@ export const Overview = (args: ButtonProps) => (
     </Flex>
   </>
 );
+
+export const SubmitButton = (args: ButtonProps) => {
+  const [isLoading, setLoading] = React.useState(false);
+
+  const submitForm = useCallback(() => {
+    setLoading(true);
+    setTimeout(() => {
+      setLoading(false);
+    }, 1000);
+  }, []);
+
+  return (
+    <>
+      <Flex flexDirection="column" marginBottom="spacingL">
+        <Flex marginBottom="spacingS">
+          <SectionHeading element="h3">
+            Submit button with loading state
+          </SectionHeading>
+        </Flex>
+        <Flex flexDirection="row" marginBottom="spacingM">
+          <Flex marginRight="spacingXs">
+            <Button
+              icon={args.icon}
+              buttonType={args.buttonType}
+              loading={isLoading}
+              disabled={isLoading}
+              onClick={submitForm}
+            >
+              Submit me
+            </Button>
+          </Flex>
+        </Flex>
+      </Flex>
+    </>
+  );
+};
```

Comment:
Could you move this story up?

We try to leave the "Overview" story as the last one always

![Screenshot 2021-09-09 at 10 49 15](https://user-images.githubusercontent.com/6597467/132654544-440479da-bdb8-4abe-b5ee-29b54ad6ca82.png)